tutorial/01_Introduction/intro.ipynb

```diff
@@ -8,7 +8,7 @@
    "source": [
```

Comment:
<p>also mention the arrays are initialized to zero</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/hpc4cmb/toast/pull/369/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>